themes/cakephp/static/css/default.css

```diff
@@ -158,6 +165,9 @@
 
 
 /* -- Sidebar -- */
+.sidebar {
+    font-family: "Open Sans", sans-serif;
```

Comment:
We definitely shouldn't.
Because of #4370 I was looking at Helvetica and not Raleway which looks old.
Thereby Open Sans shouldn't be used and we should keep Raleway.